tools/generatechangelog/generatechangelog.go

```diff
@@ -0,0 +1,282 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"text/template"
+	"time"
+
+	"github.com/go-git/go-git/v5"
```

Comment:
We should use GitHub's API instead for everything. See https://github.com/FerretDB/gh for some helpers, and please use `github.com/google/go-github/v57/github` for the rest

Reply:
refactored to use `go-github`